src/store/flow/block-types/BaseBlock.ts

```diff
@@ -35,7 +34,7 @@ export const actions = {
         }, {root: true}),
       ],
       tags: [],
-      vendor_metadata: {},
+      vendor_metadata: await dispatch('initiateExtraVendorConfig'),
```

Comment:
Should this be `vendor_metadata.config` as per https://miro.com/app/board/o9J_lXulmu4=/?moveToWidget=3458764528986875247&cot=14 ?